nemo/lightning/pytorch/plugins/data_sampler.py

```diff
@@ -3,6 +3,16 @@
 import pytorch_lightning as pl
 from torch.utils.data import DataLoader
 
+try:
+    from megatron.core.num_microbatches_calculator import (
+        get_current_global_batch_size,
+        get_num_microbatches,
+        update_num_microbatches,
+    )
+
+except (ImportError, ModuleNotFoundError):
+    from apex.transformer.pipeline_parallel.utils import get_current_global_batch_size, get_num_microbatches
```

Comment:
Can we move the imports back to where they were? We do the imports in-line to support configuring things locally without requiring all dependencies needed to run

Reply:
Could you take a look again? I have made the changes.